indico/modules/events/papers/controllers/management.py

```diff
@@ -71,3 +75,38 @@ def _process_PUT(self):
     def _process_DELETE(self):
         set_reviewing_state(self.event_new, request.view_args['reviewing_type'], False)
         return jsonify_data(flash=False, html=_render_paper_dashboard(self.event_new))
+
+
+class RHScheduleCFP(RHManagePapersBase):
+    """Schedule the call for papers"""
```

Comment:
Not a strong opinion on this, but I tend to skip docstrings for obvious classes and functions. In this case I think the class name is explicit enough not to require an explanation.